jobtracker.go

```diff
@@ -417,7 +432,14 @@ func (jt *jobTracker) fetchJobDetails(id string) (jobDetail, error) {
 	url := fmt.Sprintf("%s/proxy/%s/ws/v1/mapreduce/jobs", jt.ps, appID)
 
 	jobs := &jobsResp{}
-	if err := getJSON(url, jobs); err != nil {
+	responseStr, err := getJSON(url, jobs)
+	var submatch = redirectRegexp.FindStringSubmatch(responseStr)
+	if len(submatch) == 2 {
+		log.Printf("Response indicated a redirect but it was not followed: `%s`\n", responseStr)
```

Comment:
probably because the redirect is accompanied by a 200 status code. we could write custom redirect logic, but it seems easier just to update the URL (also requires half as many requests). it may be vaguely related to https://issues.apache.org/jira/browse/YARN-3526